pkg/subctl/cmd/join.go

```diff
@@ -103,6 +106,12 @@ func addJoinFlags(cmd *cobra.Command) {
 		"List of domains to use for multicluster service discovery")
 	cmd.Flags().StringSliceVar(&imageOverrideArr, "image-override", nil,
 		"Override component image")
+	cmd.Flags().BoolVar(&healthCheckEnable, "health-check-enable", true,
+		"Enable Gateway health check")
+	cmd.Flags().Uint64Var(&healthCheckInterval, "health-check-interval", 1,
+		"The interval in seconds in which health check packets will be send")
+	cmd.Flags().Uint64Var(&healthCheckMaxPacketLossCount, "health-check-timeout", 15,
+		"The maximum packet that can be lost after which health checker will mark a connection as down")
```

Comment:
Please ensure the description messages match the other places above.

Reply:
Done.